ios/RNTPTDocumentView.m

```diff
@@ -1294,7 +1294,7 @@ - (NSString *)exportAnnotationsWithOptions:(NSDictionary *)options
     return nil;
 }
 
-- (void)importAnnotations:(NSString *)xfdfString
+- (NSMutableArray<NSDictionary *> *)importAnnotations:(NSString *)xfdfString
```

Comment:
General suggestion to return the immutable version.
Should also specify `nullable` if the method can return `nil`.
```suggestion
- (nullable NSArray<NSDictionary *> *)importAnnotations:(NSString *)xfdfString
```